src/frontend/utils/generateOnStageRequestNotificationMessage.ts

```diff
@@ -0,0 +1,60 @@
+import { defineMessages, IntlShape } from 'react-intl';
+import { Video } from 'types/tracks';
+
+const messages = defineMessages({
+  onePersonAsking: {
+    defaultMessage: '{name} wants to go on stage.',
+    description:
+      'Notification message displayed when one person wants to go on stage.',
```

Comment:
In his own context, this util is not related to a notification.
```suggestion
      'Message displayed when one person wants to go on stage.',
```

Reply:
done